packages/manager/src/features/Volumes/Drawers/VolumeDrawer/VolumeSelect.tsx

```diff
@@ -47,6 +32,15 @@ export const VolumeSelect = (props: Props) => {
 
   return (
     <Autocomplete
+      disabled={disabled}
```

Comment:
If we retain API filtering, we should add this prop

```suggestion
      filterOptions={(options, state) => options
```

to the Autocomplete to ensure it doesn't do any client side filtering